build.gradle

```diff
@@ -47,22 +47,33 @@ cargo {
     }
 }
 
-azurewebapp {
-    deploymentType = 'wardeploy'
+azureWebApp {
     resourceGroup = "${System.env.WEBAPP_RESOURCE_GROUP}"
     appName = "${System.env.WEBAPP_NAME}"
     pricingTier = "S2"
-    authFile = file('/Users/farlen/stuff/my2.azureauth')
-    // if 'target' is not specified, dafault output of the 'war' plugin will be used
-//    target = file('C:/stuff/azure-gradle-plugins.4/samples/todo-app-java-on-azure.appservice-on-linux/build/libs/todoapp.war')
-    appServiceOnLinux = {
+    region = "southcentralus"
+
+    appService = {
+        type = 'linux'
         runtimeStack = 'TOMCAT 9.0-jre8'
-//        urlPath = 'todoapp'
     }
-//    appServiceOnWindows = {
-//        javaWebContainer = "tomcat 8.5"
-//        javaVersion = "1.8.0_102"
-//        urlPath = 'todoapp'
-//    }
+
+/*    appService = {
+        type = 'windows'
+        javaWebContainer = "tomcat 8.5"
+        javaVersion = "1.8.0_102"
+    }*/
+
+    authentication = {
+        type = "file"
+        file = "<path_to_auth_file>"
```

Comment:
Please add in readme or any other instruction to let user know this field need change

Reply:
Done, thank you.